projects/plugins/jetpack/_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -2545,6 +2545,13 @@ public static function get_updateable_data_list( $selector = '' ) {
 				'validate_callback' => __CLASS__ . '::validate_boolean',
 				'jp_group'          => 'related-posts',
 			),
+			'append_to_posts'                      => array(
+				'description'       => esc_html__( 'Automattically display related posts to the bottom of each one of your posts', 'jetpack' ),
```

Comment:
Nit: we only need one `t` in this context! 😁
I also think it would be better to use "to" instead of "at".
```suggestion
				'description'       => esc_html__( 'Automatically display related posts at the bottom of each one of your posts', 'jetpack' ),
```

Reply:
Done 8c0ced8298baab03abc66b87f851b70f7a8a3abc